docs/src/modules/brandingTheme.ts

```diff
@@ -456,6 +456,17 @@ export function getThemedComponents(): ThemeOptions {
       MuiButtonBase: {
         defaultProps: {
           disableTouchRipple: true,
+          disableRipple: true,
+        },
+        styleOverrides: {
+          root: ({ theme }) => ({
+            transition: 'all 100ms ease-in',
+            '&:active': { transform: 'scale(.98)' },
+            '&:focus-visible': {
+              outline: `3px solid ${alpha(theme.palette.primary[500], 0.5)}`,
+              outlineOffset: '2px',
+            },
```

Comment:
Oh, on ButtonBase, hum, this is ambitious.

Reply:
I've circled around both the marketing pages and the documentation, and aside from the instance in your PR, nothing crazy stood out. In any case, do let me know if you find any further regressions! 👍 Many buttons/links were missing proper focus state treatment prior to this.

Reply:
> Many buttons/links were missing proper focus state treatment prior to this.

True, this might make for the best tradeoff. We never thought about this use case for ButtonBase when introducing it 😄